src/routes/TraderForm/Main/PoolHeader/index.tsx

```diff
@@ -55,7 +56,13 @@ export const PoolHeader: React.FunctionComponent<PoolHeaderProps> = () => {
         <VariableBox>
           <LabelTokenTypography
             colorToken="lavenderWeb"
-            // differenceValue={-2} //TODO Alex
+            differenceValue={
+              variableRateInfo24hAgo.status === 'success' && variableRateInfo.status === 'success'
```

Comment:
why not move this to selector? This entire logic can live into a selector that returns either a formatted number or undefined. 

Might be best place for it.

Reply:
[753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)